docs/repositories-configure/managing-branches.md

```diff
@@ -24,7 +24,7 @@ To change the main branch of your repository or enable analysis on other branche
 
 This page also displays the [code quality grade](../faq/code-analysis/which-metrics-does-codacy-calculate.md) for each enabled branch.
 
-![Managing branches](images/managing-branches-sh.png)
+![Managing branches](images/managing-branches-sh.png)<!--TODO PLUTO-532 Update screenshot-->
```

Comment:
We don't need to update this screenshot, since the updates to the Gates and Goals won't be reflected on Self-hosted.

Reply:
Thanks for the heads-up, @prcr! 🙏